scripts/trilinos-integration/README.md

```diff
@@ -0,0 +1,59 @@
+![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)
+
+# Kokkos: Promotion Test Debugging
+
+This explains the use (and basic implementation details) of promotion testing
+new Kokkos and Kokkos Kernels branches. We first introduce a test for first
+validating an existing Trilinos branch that all tests should be passing on (usually the develop, master, or kokkos-promotion branch of Trilinos).
+After validating a clean promotion branch, we show how to test the same Trilinos branch
+with an updated Kokkoks and Kokkos Kernels.
+
+# Clean Promotion Test
+There is a script called `clean_promotion_test` in `scripts/trilinos-integration` that tests a self-contained
+Trilinos branch with its default included Kokkos and Kokkos Kernels packages.
+The script takes two arguments:
+````
+./clean_promotion_test <ENV_FILE> <TRILINOS_ROOT>
+````
+For running CUDA tests on the platform called White, e.g.
+````
+./clean_promotion_test white_cuda_env.sh $TRILINOS_ROOT
+````
+Based on the Trilinos path and ENV file, a unique hash `X` is generated.
+A CMake configuration is then run in the folder `trilinos-promotion-X`.
```

Comment:
when I tested this, I got a directory called promotion-test-X

Reply:
Yes, you did. I fixed the docs. I also changed the "clean" tests to run in `clean-test-X`